codebuild/spec/buildspec_unit_nix.yaml

```diff
@@ -0,0 +1,167 @@
+---
+version: 0.2
+env:
+  shell: bash
+  variables:
+    NIXDEV_ARGS: --max-jobs auto
+batch:
+  build-graph:
+    # Cache job for x86
+    - identifier: nixCache_x86_64
+      comment: identifiers can not contain dashes
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild:latest
+        variables:
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-x86-64?region=us-west-2
+    # Cache job for aarch64
+    - identifier: nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        type: ARM_CONTAINER
+        variables:
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-aarch64?region=us-west-2
+    # Openssl30 x86
+    - identifier: UnitOpenssl30_x86_64
+      depend-on:
+        - nixCache_x86_64
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild:latest
+        privileged-mode: true
+        variables:
+          NIX_CACHE_BUCKET: s3://s2n-tls-nixcachebucket-x86-64?region=us-west-2
```

Comment:
I wonder if you could define env vars for these buckets higher up so that here you could just write:
```
          NIX_CACHE_BUCKET: X86_NIX_CACHE_BUCKET
```

Reply:
I tried this with `NIX_CACHE_BUCKET: $X86_NIX_CACHE_BUCKET`, which didn't work, b/c it doesn't look like CodeBuild is doing variable expansion.  I think the above would treat "X86_NIX_CACHE_BUCKET" as the string value and also not expand it?